mycore-mods/src/main/java/org/mycore/mods/identifier/MCRMODSURNPIMetadataService.java

```diff
@@ -27,17 +27,17 @@
 import org.mycore.datamodel.metadata.MCRObject;
 import org.mycore.mods.MCRMODSWrapper;
 import org.mycore.pi.MCRPersistentIdentifier;
-import org.mycore.pi.MCRPersistentIdentifierMetadataManager;
+import org.mycore.pi.MCRPIMetadataService;
 import org.mycore.pi.exceptions.MCRPersistentIdentifierException;
 import org.mycore.pi.urn.MCRDNBURNParser;
 import org.mycore.pi.urn.MCRUniformResourceName;
 
-public class MCRMODSURNPersistentIdentifierMetadataManager
-    extends MCRPersistentIdentifierMetadataManager<MCRUniformResourceName> {
+public class MCRMODSURNPIMetadataService
```

Comment:
This class contains a "PI" in its name (which I would prefer), but  "MCRMODSDOIMetadataService" not.